src/Config/BaseConfig.php

```diff
@@ -290,12 +290,12 @@ public function getEnvKbcUrl(): string
         return (string) $env;
     }
 
-    public function getDataTypeSupport(): string
+    public function getDataTypeSupport(): DatatypeSupport
     {
         $env = getenv('KBC_DATA_TYPE_SUPPORT');
         if (!$env) {
-            return 'none';
+            return DatatypeSupport::NONE;
         }
-        return $this->getStringValue(['storage', 'output', 'data_type_support'], (string) $env);
+        return DatatypeSupport::from($this->getStringValue(['storage', 'output', 'data_type_support'], (string) $env));
```

Comment:
tady by to chtělo asi odchytit vyhodit user exception, když to user přepíše v configu na non enum value

Reply:
tak tam može byt tryFrom aby to vrátilo null :thinking:

Reply:
jo, to pak to fallbackne na ten default no.

Reply:
apdejtnuto

Reply:
akorát bude potřeba vydat novů major verzi teda.

Reply:
Já bych spíš ten release 10 zrušil, tam bylo víc chyb.